packages/metadata-common/src/AddMetadataButton.tsx

```diff
@@ -35,7 +35,7 @@ export const METADATA_HEADER_POPPER_CLASS = 'elyra-metadataHeader-popper';
 export interface IAddMetadataButtonProps {
   schemas?: IDictionary<any>[];
   addMetadata: (schema: string) => void;
-  // Optional string to append to the schema display name
+  // Optional string to append to the schema title
   schemaType?: string;
```

Comment:
`schemaType` doesn't seem very indicative of what the comment says that it's for.  Seems like `additionalTitle` or `titleContext` or something more descriptive seems more appropriate since, iiuc, this is additional information like "runtime configuration" that follows the `title` of "Kubeflow Pipelines".

Reply:
I agree that titleContext fits better. Will add that on.